p8e-shared/src/main/kotlin/io/provenance/p8e/shared/service/AffiliateService.kt

```diff
@@ -296,6 +296,11 @@ class AffiliateService(
                         require (response.isAcknowledged) { "ES index creation of $it was not successful" }
                     }
                 }
+
+                if (jwt != null && identityUuid != null) {
+                    keystoneService.registerKey(jwt, signingPublicKey.publicKey, ECUtils.LEGACY_DIME_CURVE, KeystoneKeyUsage.CONTRACT)
```

Comment:
I just copied this block from the other save function, this gets removed in the branch that uses the new object store (since that is where the oauth authentication removal is as well), so I just copied it all to keep it consistent until it all gets merged together